Eradication/MigrationInfoVector.cpp

```diff
@@ -91,14 +422,21 @@ namespace Kernel
         // --- We only want to save raw migration rates for female 
         // --- vectors, because male vector rates do not get modified
         // ----------------------------------------------------------
+
```

Comment:
SaveRawRates() - Since CalcualteRates() does nothing, I don't see that SaveRawRates() ever gets called.  Maybe this should be a release_assert( false ); or don't override it at all.

Hmm.  I must be missing something.  If SaveRawRates() doesn't get called, then m_RawMigrationRateFemale should be empty.

Reply:
we've talked about this, but now SaveRawRates does stuff!